packages/builder/src/dataBinding.js

```diff
@@ -813,6 +815,26 @@ export const getActionBindings = (actions, actionId) => {
   return bindings
 }
 
+/**
+ * Gets all device bindings for embeds.
+ */
+const getEmbedBindings = () => {
+  let bindings = []
+  const safeEmbed = makePropSafe("embed")
+
+  bindings = [
+    {
+      type: "context",
+      runtimeBinding: `${safeEmbed}`,
+      readableBinding: `ParentWindow`,
+      category: "Embed",
+      icon: "DistributeVertically",
+      display: { type: "object", name: "Parent Window" },
```

Comment:
NAB: The display name is optional here and overrides the readable binding, so we may as well remove it and just let it use the readable binding (which I'd also put a space in). Icon looks grand to me.

Reply:
Removed display and just kept `ParentWindow` as one because the binding looks cleaner